cmd/publishing-bot/publisher.go

```diff
@@ -404,7 +404,12 @@ func (p *PublisherMunger) publish(newUpstreamHeads map[string]plumbing.Hash) err
 			if !ok {
 				return fmt.Errorf("no upstream branch %q found", branchRule.Source.Branch)
 			}
-			if err := os.WriteFile(path.Join(path.Dir(dstDir), publishedFileName(repoRules.DestinationRepository, branchRule.Name)), []byte(upstreamBranchHead.String()), 0o644); err != nil {
+			if err := os.WriteFile(
```

Comment:
is the indentation here ok? or only looking weird to me

Reply:
fixed the indentation